index.html

```diff
@@ -20,33 +20,33 @@
         </div>
         <div class="cal-body">
             <div class="row">
-                <button  class="key operator2" onclick="reset()">AC</button>
-                <button  class="key operator2" onclick=" toggleNegate()" >+/-</button>
-                <button  class="key operator2 " onclick="inputOperator('%')">%</button>
-                <button  class="key operator " onclick="inputOperator('÷')">÷</button>
-            </div>
+                <button  class="key operator2" >AC</button>
+                <button  class="key operator2"  >+/-</button>
+                <button  class="key operator2" >%</button>
```

Comment:
operator 2 is not very meaningful name
If we want to  only apply styling from this class name then we can add some other meaningful name i.e. yellow-box, light-grey-box, dark-grey-box etc